region/wreckedship/main.json

```diff
@@ -1995,7 +2009,25 @@
                     "h_canDestroyBombWalls",
                     "f_DefeatedPhantoon"
                   ],
-                  "note": "The path can't be passed unless the Workrobot is activated (so Phantoon dead)"
+                  "note": "By normal means, the path can't be passed unless the Workrobot is activated (so Phantoon dead)"
+                },
+                {
+                  "name": "WS East Supers Robot Clip (Left to Right)",
+                  "notable": true,
+                  "requires": [
+                    "Morph",
+                    "h_canDestroyBombWalls",
+                    "canTrickyJump"
+                  ],
+                  "note": [
+                    "Kago through the Workrobot by jumping, pressing down, then morphing and moving right just before hitting the robot.",
```

Comment:
Kago tech would make sense to add now.  Its only lacking the logically required situations for it to be made a tech, and here is one.